### PR TITLE
feat(sdk): add some missing functions to math

### PR DESCRIPTION
**Pull request by mortax**
Wing doesn't include `sign` or any standard logarithm functions (i.e. `log`, `log12`, or `log10`), which are handy -- and shouldn't require escaping to JavaScript.

This PR adds them to `math`.

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Wing Cloud Contribution License](https://github.com/winglang/wing/blob/main/CONTRIBUTION_LICENSE.md)*.

### CODE REVIEW

**Review by eladb — APPROVED** [review]
Awesome stuff!

File: examples/tests/sdk_tests/math/sign.test.w [hunk 1]
```diff
@@ -0,0 +1,12 @@
+bring math;
+
+let x = 3;
+let y = 5;
+
+assert(math.abs(y - x) == 2);
```
Comment (staycoolcall911) [inline examples/tests/sdk_tests/math/sign.test.w:6]: I think you wanted to test `math.sign()` here instead of `math.abs()`

**Review by staycoolcall911 — CHANGES_REQUESTED** [review]
Looks great, there's 1 typo in sign.test.w

File: examples/tests/sdk_tests/math/sign.test.w (hunk 1 above)
Comment (mortax) [reply examples/tests/sdk_tests/math/sign.test.w:6]: well that's embarrassing! it looks like i prepared to write the new test but didn't. 🤦‍♂️ fixed!

Comment (staycoolcall911) [reply examples/tests/sdk_tests/math/sign.test.w]: Nice, thank you! I'll just check that it runs well on aws and approve

**Review by staycoolcall911 — APPROVED** [review]

**Pull request merged**